TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/groups/ZeroRing.java

```diff
@@ -64,22 +66,17 @@ public ZeroRing(MecanumDriveSubsystem drive, WobbleSubsystem wobbleSystem, Shoot
                 new TrajectoryFollowerCommand(drive, traj2),
                 new TurnCommand(drive, Math.toRadians(10)),
                 new RapidFireCommand(shooter),
-                new ParallelDeadlineGroup(
-                        new Com_PutDown(wobbleSystem),
-                        new TurnCommand(drive, Math.toRadians(-10))
-                ),
+                new Com_PutDown(wobbleSystem),
                 new InstantCommand(shooter::stop, shooter),
                 new TrajectoryFollowerCommand(drive, traj3),
-                new TrajectoryFollowerCommand(drive, traj4),
                 new InstantCommand(wobbleSystem::closeGrabber, wobbleSystem),
                 new WaitCommand(1000),
-                new Com_PickUp(wobbleSystem),
-                new TrajectoryFollowerCommand(drive, traj5),
+                new TrajectoryFollowerCommand(drive, traj4),
                 new TurnCommand(drive, Math.toRadians(180)),
-                new Com_PutDown(wobbleSystem),
                 new InstantCommand(wobbleSystem::openGrabber, wobbleSystem),
                 new WaitCommand(500),
-                new Com_PickUp(wobbleSystem)
+                new Com_PickUp(wobbleSystem),
+                new TrajectoryFollowerCommand(drive, traj5)
```

Comment:
Put these in a parallel command group.
```suggestion
                new ParallelCommandGroup(
                    new Com_PickUp(wobbleSystem),
                    new TrajectoryFollowerCommand(drive, traj5)
                )
```